ReactWindows/ReactNative/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -335,6 +308,31 @@ public void SetEditable(PasswordBox view, bool editable)
             view.IsEnabled = editable;
         }
 
+        /// <summary>
+        /// Sets whether the view is a tab stop.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="isTabStop">
+        /// <code>true</code> if the view is a tab stop, otherwise <code>false</code>.
+        /// </param>
+        /// 
+        [ReactProp("isTabStop")]
+        public void SetIsTabStop(PasswordBox view, bool isTabStop)
+        {
+            view.IsTabStop = isTabStop;
```

Comment:
>IsTabStop [](start = 17, length = 9)

Consider adding a comment which states that the control is not focusable at all when `IsTabStop` is `false`. I don't think this effect is obvious from the name alone.

Reply:
fixed

---
In reply to: [156584942](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156584942) [](ancestors = 156584942)